src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,22 @@
+import styles from "./Loader.module.scss";
+
+export enum LoaderSize {
+  s = "s",
+  m = "m",
+  l = "l",
+}
+
+export type LoaderProps = {
+  loading?: boolean;
+  size?: LoaderSize;
+  className?: string;
+};
+
+export const Loader = ({
+  loading = true,
+  size = LoaderSize.m,
+}: LoaderProps) => {
+  return loading ? (
+    <div className={`${styles.loader} ${styles.size_m}`}></div>
+  ) : null;
```

Comment:
1. ты не используешь проп `size`. должно быть ``styles[`size_${size}`]``
2. в таких случаях стоит использовать либу [classnames](https://www.npmjs.com/package/classnames), либо, если не хочется тащить в проект лишнюю либу, можно попробовать написать что-то своё, например:
```tsx
const classnames = (...args: any[]): string => args.filter(arg => typeof arg === 'string').join(' ');
// использование
<div className={classnames(styles.loader, styles[`size_${size}`])} />
```
вот тебе кстати и пример оправданного использования  `any` - можно прокинуть в функцию массив всего, чего угодно, нам не за чем это запрещать, всё равно мы отфильтруем только строчные значения
3. я бы использовал тут ранний `return`, с ним более читабельно, чем с тернарным оператором
```suggestion
  if (!loading) {
    return null;
  }

  return <div className={`${styles.loader} ${styles.size_m}`}></div>
```